todo-app/src/App.js

```diff
@@ -1,13 +1,22 @@
-import React, { Component } from 'react';
+import React from 'react'
+import ToDo from './ToDo.js'
+import {ToDos} from './toDos-DB.js'
 
-class App extends Component {
+export default class App extends React.Component {
+  renderToDos(toDos) {
+    return toDos.map((toDo, index) => {
```

Comment:
Instead of rendering the `toDos` directly from the import, you should have set them in `App` state and use them from there. For example:

```js
export default class App extends React.Component {
  constructor(props) {
    super(props);
    this.state = {
      todos: ToDos
    };
  }

  // we don't need to modify renderToDos

  render() {
    return (
      <div>
        <h1>ToDo List</h1>
        {this.renderToDos(this.state.todos)}
      </div>
    )
  }
}
```

The will help us in updating and re-rendering. When for example the status of a todo change and we update the state, then React will now that it has to render the components that are affected by this change. You now have to do some tricks to accomplish this, which should be avoided (see `ToDoStatus ` comments).